uplc.md

```diff
@@ -56,71 +74,186 @@ module UNTYPED-PLUTUS-CORE-GRAMMAR
                   | "(" "lam" Var Term ")"              // lambda abstraction
                   | "(" "delay" Term ")"                // delay execution of a term
 
+   syntax TermList ::= ".TermList" | Term "" TermList
```

Comment:
I see you left a "" here. I think this should be removed..

Reply:
This is part of the syntax. It´s juxtaposition. But I am experimenting with ways to handle this. Using syntactic lists generates ambiguities.